sites/page-details/index.json-ld

```diff
@@ -82,5 +83,346 @@
+					"Pour afficher le bouton \"Partagez cette page\", dans l'en-tête de la page, définissez la propriété <code>share</code> à <code>true</code>."
+				]
+			},
+			"sample": {
+				"en": [
+					{
+						"@type": "source-code",
+						"description": "Code sample:",
+						"code": "\"feedback\": true, \n\"share\": true"
+					}
+				],
+				"fr": [
+					{
+						"@type": "source-code",
+						"description": "Exemple de code:",
+						"code": "\"feedback\": true, \n\"share\": true"
+					}
+				]
+			}
+		}
+	],
+	"iteration": [
+		{
+			"@id": "_:iteration_pd_3",
+			"name": "Page details - Iteration 3",
+			"date": "2023-03",
+			"detectableBy": "section.pagedetails",
+			"predecessor": "_:iteration_pd_2",
+			"fixes": [
+				"Changed <footer> to <section> as <footer> element does not provide a semantic break."
+			],
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-v3-en.html#full"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Exemple de code",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "samples/page-details-v3-fr.html#full"
+					}
+				}
+			]
+		},
+		{
+			"@id": "_:iteration_pd_2",
+			"name": "Page details - Iteration 2",
+			"date": "2022-05",
+			"detectableBy": "footer.pagedetails",
+			"predecessor": "_:iteration_pd_1",
+			"successor": "_:iteration_pd_3",
+			"breaking": [
+				"Wrapping <div> changed to <footer>.",
+				"Changed grid sizes for RAP and Share this page.",
+				"RAP is now loaded through Data-AJAX with a fallback for basic mode."
+			],
+			"additions": [
+				"New invisible heading for accessibility."
+			],
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "deprecated/page-details-v2-en.html"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Exemple de code",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "deprecated/page-details-v2-fr.html"
+					}
+				}
+			]
+		},
+		{
+			"@id": "_:iteration_pd_1",
+			"name": "Page details - Iteration 1",
+			"date": "2017-10",
+			"detectableBy": "div.pagedetails",
+			"successor": "_:iteration_pd_2",
+			"assets": [
+				{
+					"@type": "source-code",
+					"@language": "en",
+					"description": "Code sample",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "deprecated/page-details-v1-en.html"
+					}
+				},
+				{
+					"@type": "source-code",
+					"@language": "fr",
+					"description": "Exemple de code",
+					"code": {
+						"@type": [ "rdf:HTML", "@id" ],
+						"@value": "deprecated/page-details-v1-fr.html"
+					}
+				}
+			]
+		}
+	],
+	"changesets": [
+		{
```

Comment:
We should add another change set for the version 1 of the page details which could be very briefly described. That way we would be able to announce it is deprecated.